fem/gridfunc.cpp

```diff
@@ -2009,7 +2009,7 @@ double GridFunction::ComputeL2Error(
       fdof = fe->GetDof();
       transf = fes->GetElementTransformation(i);
       shape.SetSize(fdof);
-      intorder = 2*fe->GetOrder() + 1; // <----------
+      intorder = 2*fe->GetOrder() + 3; // <----------
```

Comment:
This was always suspicious, and still is, but I'm not sure what to suggest instead...